internal/area/area.go

```diff
@@ -42,16 +42,29 @@ func FromSize(size image.Point) (image.Rectangle, error) {
 // specified percentage of its width. The percentage must be in the range
 // 0 <= heightPerc <= 100.
 // Can return zero size areas.
-func HSplit(area image.Rectangle, heightPerc int) (top image.Rectangle, bottom image.Rectangle, err error) {
+//
+// The fixed value must by in the range -1 <= heightFixed.
+// If fixed value is -1 (default value), heightPerc is used instead.
+func HSplit(area image.Rectangle, heightPerc int, heightFixed int) (top image.Rectangle, bottom image.Rectangle, err error) {
```

Comment:
One of good principles to follow when it comes to code readability is to keep functions focused on just one thing. The HSplit() and VSplit() functions perform relative splits based on percentage. Adding another argument that changes their behavior makes both the calling code and these functions harder to understand.

Normally the approach would be to implement two new functions that perform fixed splits, however it looks like we are lucky and those already exist:

https://github.com/mum4k/termdash/blob/0e5242ca6f8386889a303c67b58a25f98aed75b6/internal/area/area.go#L81-L86

https://github.com/mum4k/termdash/blob/0e5242ca6f8386889a303c67b58a25f98aed75b6/internal/area/area.go#L104-L109

Would you mind taking a look if they can be used out of the box or if they need to be modified? Once they are ready for us, we can modify the calling code to choose which function to call based on which option was provided.

Reply:
These worked great. Dropped them in and it just works